OpenRA.Game/Widgets/Widget.cs

```diff
@@ -204,6 +206,8 @@ public Widget(Widget widget)
 			IgnoreChildMouseOver = widget.IgnoreChildMouseOver;
 			IgnoreMouseOver = widget.IgnoreMouseOver;
 
+			ChromeMetrics.TryGet<string>("DefaultCursor", out defaultCursor);
```

Comment:
Should we throw an exception here if the get fails, or fall back to a default value? (The former is probably better, unless it is intentional that we ignore failure.)

Reply:
Went with throwing for consistency.